README.md

```diff
@@ -1,36 +1,60 @@
 # Structure
 
-Pds stands for Product Design System.  The original intent was to rename `structure` to Pds because the developer at the time thought it was more clear.  Unfortunately, that renaming never happened so there are two names.
+## 🚨 NOTICE 🚨
 
-This is a mono repo with the idea that the Ember, React, etc projects scattered around Hashi could all eventually rely on a single source for design components.  Currently there is only `pds-ember`.
+Currently this project in a "maintained sunsetting" mode. This means we're starting to remove deprecated/unused components from its codebase with the intent to completely remove as dependency from our products.
 
-## Getting Started
-To contribute to the `pds-ember` package (currently the only package in this mono repo) please see the [Contributing](packages/pds-ember/CONTRIBUTING.md) guide for details.
+But don't worry, if you need to use the old project, there are a couple of "frozen in time" branches that you can use (and potentially even release from, if you need). See *"Which version to use?"* below.
+
+### Which version to use?
+
+The current version of this package is the one used for the "managed sunsetting" of the library from our products. As such, in time some components are deprecated and then removed from the repository, and some surgical patches to the code are done when necessary to facilitate the migration to the new UI library ([HashiCorp Design System components](https://github.com/hashicorp/design-system)).
+
+**If for any reasons you need to stay on an older version**, you can:
+- use the package version `0.6.4` to stay "frozen in time" at the moment in which the "maintained sunsetting" phase has started.
+- use the package version `1.x.x` to stay on a release path that sees only minor patches but not components removal
+  - in this case, if you need to make changes, use the branch `master-version-1.x` and release to NPM from that branch
+
+---
 
+*Old introduction to the project, left for reference:*
 
-## Creating + Consuming a Release
+> Pds stands for Product Design System.  The original intent was to rename `structure` to Pds because the developer at the time thought it was more clear.  Unfortunately, that renaming never happened so there are two names.
+> This is a mono repo with the idea that the Ember, React, etc projects scattered around Hashi could all eventually rely on a single source for design components.  Currently there is only `pds-ember`.
 
-### Update Version
-1. `yarn workspace @hashicorp/pds-ember version patch` to bump the version by `0.0.1`
+## Getting started
+To contribute to the `pds-ember` package (currently the only package in this mono repo) please see the [Contributing](packages/pds-ember/CONTRIBUTING.md) guide for details.
+
+## Creating + Consuming a release
+
+### Update version
+1. `yarn workspace @hashicorp/pds-ember version patch` to bump the version by `2.0.2`
```

Comment:
```suggestion
1. `yarn workspace @hashicorp/pds-ember version patch` to bump the version by `0.0.1`
```

I think technically the previous version of this was more accurate as running this command increments the patch version by one

Reply:
ahh now I understand what that meant. I'll re-write that paragraph to make it more clear.

Reply:
fixed